airflow/config_templates/airflow_local_settings.py

```diff
@@ -104,6 +108,26 @@
     }
 }
 
+DEFAULT_DAG_PARSING_LOGGING_CONFIG = {
+    'handlers': {
+        'processor_manager': {
+            'class': 'logging.handlers.RotatingFileHandler',
+            'formatter': 'airflow',
+            'filename': LOG_PROCESSOR_MANAGER_LOCATION,
+            'mode': 'a',
```

Comment:
n00b qq: are these mode options defined in `RotatingFileHandler`?

Reply:
sry I didn't get ur question... Are you asking what are these options for? That is the file opening mode, set to 'a' so we keep appending to the file( so we don't overwrite the log if we restart the scheduler or so). 'a' is indeed the default mode but just wanted to keep it here to make it more clear